week_1/Project/youtube-fe/styles/header.css

```diff
@@ -0,0 +1,123 @@
+.header {
+    height: 55px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #fff;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 100;
+}
+
+.left-section {
+    display: flex;
+    align-items: center;
+}
+
+.hamburger-menu {
+    height: 24px;
+    margin: 0px 20px 0px 24px;
+}
+
+.youtube-logo {
+    height: 22px;
+}
+
+.middle-section {
+    flex: 1;
+    margin: 0px 35px 0px 70px;
+    max-width: 650px;
+    display: flex;
+    align-items: center;
+}
+
+.search-bar {
+    flex: 1;
+    height: 34px;
+    padding-left: 10px;
+    font-size: 16px;
+    border-width: 1px;
+    border-style: solid;
+    border-top-left-radius: 20px;
+    border-bottom-left-radius: 20px;
+    width: 0;
+}
+
+.search-bar::placeholder {
+    font-family: Arial;
+    font-size: 16px;
+}
+
+.search-button {
+    height: 38px;
+    width: 60px;
+    border-style: solid;
+    border-color: gray;
+    border-width: 1px;
+    margin-right: 20px;
+    border-top-right-radius: 20px;
+    border-bottom-right-radius: 20px;
+    /* Currently there is no other way I can design it without using negative margin, so using this until I learn a better way */
+    margin-left: -1px;
+}
+
+.search-icon {
+    height: 25px;
+}
+
+.voice-search-button {
+    height: 40px;
+    width: 40px;
+    border: none;
+    border-radius: 20px;
```

Comment:
@codal-kcolaco is you are giving `border: none;` then what is use of giving `border-radius: 20px;`

Reply:
@abhi-codal Yes was experimenting with the element and forgot to remove the border. Have updated that in code.